test/testcafe/spec/ChallengeOktaVerifySSOExtensionView_spec.js

```diff
@@ -12,7 +13,12 @@ const credentialSSOExtensionMock = RequestMock()
   .onRequestTo(/idp\/idx\/introspect/)
   .respond(identifyUserVerificationWithCredentialSSOExtension)
   .onRequestTo(verifyUrl)
-  .respond(identify);
+  .respond((req, res) => {
+    return new Promise((resolve) => setTimeout(function() {
+      res.setBody(identify);
+      resolve(res);
+    }, Constants.TESTCAFE_DEFAULT_AJAX_WAIT + 1000));
```

Comment:
Why is this wait needed?

Reply:
Add comments at line 51. 

Here is what testcafe does (my understanding)

1. `await ssoExtensionPage.navigateToPage();`  --->  wait **3s** because there is ajax in flight.
2. continue next expression.

(3s is hard-code by testcafe)

Let's say the ajax takes less than 3s, then while testcafe is wait (step 1), ajax finished and widget will proceed to next view (success view for instance), so when it runs into step 2, widget already at success view hence no way to verify ssoExtensionPage.
The trick it did before is to error out the ajax (remove `method` from API mock hence `idx.proceed` will show error therefore we stuck on the ssoExtensionPage and can verify page elements.